installer-for-dhs/build.gradle

```diff
@@ -53,3 +54,27 @@ task installDhfIntoDhs(type: Exec) {
 }
 installDhfIntoDhs.dependsOn jar
 
+
+publishing {
+    publications {
+        maven(MavenPublication) {
+            artifact("build/libs/marklogic-data-hub-$version-installer"+".jar")
+            artifactId = 'marklogic-data-hub-installer'
```

Comment:
This isn't a good choice, because it sounds like it's a generic DHF installer. But it's specifically for DHS. Granted, the line above should ideally have "installer-for-dhs" in it as well. But if this is only being published internally, "installer-for-dhs" should be used as the artifactId.

Reply:
fixed to artifact name installer-for-dhs